zmq-eio/test/dune

```diff
@@ -0,0 +1,11 @@
+(executable
+ (name test)
+ (libraries zmq-eio ounit2 eio eio_main))
+
+(rule
+ (alias runtest)
+ (deps
+  (:test test.exe))
```

Comment:
`run` implicitly creates a dependency on `test`, so this could be skipped.

Reply:
I'm not sure I understand. Can the rule be skipped altogether?